test/check-application

```diff
@@ -14,41 +14,42 @@ import testlib
 
 
 class TestApplication(testlib.MachineCase):
+    def createCertificate(self, nickName):
+        m = self.machine
+
+        m.execute("selfsign-getcert request -d /etc/pki/nssdb -n {0}".format(nickName))
+
+    def testEmptyState(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("systemctl stop certmonger.service")
+
+        self.login_and_go("/certificates")
+        b.wait_text(".pf-c-empty-state h1", "certmonger.service is Not Active")
+
+        b.click(".pf-c-empty-state button:contains(Start certmonger.service)")
```

Comment:
Similar to podman, I think a better label is: 
`Start certmonger`

Service is already mentioned in the header, and it's a bit of an implementation detail.

Reply:
Fixed